core/src/software/aws/toolkits/core/credentials/sso/SsoCredentialProvider.kt

```diff
@@ -27,7 +28,7 @@ class SsoCredentialProvider(
     private val ssoRole: String,
     private val ssoClient: SsoClient,
     private val ssoAccessTokenProvider: SsoAccessTokenProvider
-) : AwsCredentialsProvider {
+) : AwsCredentialsProvider, SdkAutoCloseable {
```

Comment:
out of interest - why `SdkAutoCloseable` instead of just the regular `AutoCloseable` ? Does the SDK do anything automatically for us here?